lib/tmp.js

```diff
@@ -97,14 +97,14 @@ function _isUndefined(obj) {
  */
 function _parseArguments(options, callback) {
   if (typeof options == 'function') {
-    return [callback || {}, options];
+    return [{}, options];
   }
 
   if (_isUndefined(options)) {
     return [{}, callback];
   }
 
-  return [options, callback];
+  return [options, callback]; 
```

Comment:
fixed, but not on this line :grin:, there was an extra WS in line 271 after the fs.closeSync(); call.